src/bundle/bundleRouter.ts

```diff
@@ -109,8 +127,20 @@ export function createBundleRouter(props: IBundleRouteProps): IBundleRouter {
           if (!cssBundle) createCSSBundle();
           cssBundle.source.modules.push(module);
         } else if (module.pkg.type === PackageType.EXTERNAL_PACKAGE && hasVendorConfig) {
-          if (!vendorBundle) createVendorBundle();
-          vendorBundle.source.modules.push(module);
+          let isMappedBundle = false;
```

Comment:
it will work only for the external packages. I think it should work against any path.

Reply:
Totally true, my bad.

Reply:
I've been checking this, and if EXTERNAL_PACKAGE check is not included, it will never create an `app.js` bundle since all will fall into this condition